README.md

```diff
@@ -1,2 +1,3 @@
-# eslint-config-qlean
-JavaScript Qlean Style - EsLint Shareable Config
+# Eslint Config
+
+JavaScript web and React Native Qlean style shareable configs
```

Comment:
I suggest to cut the title, write something like "Qlean extensions for Airbnb config" and add links to actual configs readmes.